subfun/getInfo.m

```diff
@@ -19,9 +18,15 @@
 % opt - options structure defined by the getOption function. Mostly used to find the
 % task name.
 
+if ~any(nargin == [4, 5, 7])
+    error('getInfo needs 4, 5 or 7 input argument to work)')
+end
+
+info = lower(info);
```

Comment:
make the function more flexible by allowing this string argument to be typed in either upper or lower case